src/main/java/dev/socketmods/socketnukes/item/block/ExplosiveBlockItem.java

```diff
@@ -0,0 +1,53 @@
+package dev.socketmods.socketnukes.item.block;
+
+import dev.socketmods.socketnukes.SocketNukes;
+import dev.socketmods.socketnukes.capability.Capabilities;
+import dev.socketmods.socketnukes.capability.exploderconfig.IConfiguration;
+import dev.socketmods.socketnukes.item.SocketItems;
+import dev.socketmods.socketnukes.registry.ExtendedExplosionType;
+import dev.socketmods.socketnukes.registry.SNRegistry;
+import net.minecraft.item.BlockItem;
+import net.minecraft.item.ItemGroup;
+import net.minecraft.item.ItemStack;
+import net.minecraft.util.NonNullList;
+import net.minecraft.util.ResourceLocation;
+import net.minecraft.util.text.ITextComponent;
+import net.minecraft.util.text.StringTextComponent;
+import net.minecraft.util.text.TranslationTextComponent;
+import net.minecraftforge.fml.RegistryObject;
+
+import javax.annotation.Resource;
+
+/**
+ * Multiple use block item.
+ * Stores and ignites different explosions depending on what was crafted, or chosen.
+ *
+ * @author Citrine
+ */
+public class ExplosiveBlockItem extends BlockItem {
+
+    public ExplosiveBlockItem() {
+        super(SNRegistry.GENERIC_EXPLOSIVE.get(), SocketItems.EXPLOSIVE_PROPERTIES);
```

Comment:
Have the constructor accept a `Item.Properties` instance and pass it to the super constructor, and move the `EXPLOSIVE_PROPERTIES` argument to the place where this constructor is called.